### PR TITLE
create the switch for the probation fees changes

### PR DESCRIPTION
**Pull request by leoapost**
**Note: Please do not merge this PR, until further notice.**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ENV=dev
-
+PROBATE_FEES_RELEASE_DATE=2017-05-01
```
Comment (zaparka) [inline .env.development]: I think that you should set this to different data so we can test it :) after it's deployed, so let say Monday next week?

Comment (leoapost) [inline .env.development]: this gets loaded only on development envs. We can se the PROBATE_FEES_RELEASE_DATE variable to any date we like during the staging deployment.

Comment (zaparka) [inline .env.development]: @leoapost How?

Comment (leoapost) [inline .env.development]: Can't we set ENV vars from the deployer?

Comment (zaparka) [inline .env.development]: Not that I know of.

**Pull request merged**